less/home-page.less

```
@@ -1,11 +1,59 @@
+//Header
+
+header {
+  display:flex;
+  align-items: center;
+  padding: 20px 0;
+  margin: 0 auto;
+  background-color: #dee2e6;
+
+  
+  
+  
+  
+  //@media (max-width: 500px) {
+    //flex-direction: column;
+    
+  //}
+  
+}
+
+.header-contents {
+  display:flex;
+  max-width: 800px;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 auto;
+  width: 100%;
+
+  h1 {
+    font-size: 4rem;
+  }
+  
+}
+
 // Home Content
 
 .home {
+  
 
   .intro {
+    display:flex;
+    flex-direction:column;
+    background-color: white;
+    padding-top: 0;
+    align-items: flex-start;
+
+    img {
+      width: 100%;
+    }
+    
 
     h2 {
       padding: 15px;
```

```suggestion
      padding: 15px 1.5%;
```
It's OK to have the top and bottom be hard coded at 15px, but left and right should be a percentage.